src/main/java/ch/njol/skript/expressions/ExprIP.java

```diff
@@ -38,7 +38,7 @@
 @Examples({"IP-ban the player # is equal to the next line",
 		"ban the IP-address of the player",
 		"broadcast \"Banned the IP %IP of player%\""})
-@Since("1.4, 2.2-dev35 (Converted to SimplePropertyExpression)")
+@Since("1.4, 2.2-dev35")

```

Comment:
Damn, what does this explain exactly?!?!

Reply:
Are you serious!?!? You just said to remove it now you're saying to add some information about it. No just stop... Like Bensku said, if someone wants to understand what was changed in the version they can go through Git to figure out what was change.

Reply:
That is worse. Since is there to show the addition version of something, not a version list. I meant there's no need to mention about this change. Because Skript is english so everyone would know that.

Reply:
So then why don't I just keep it as (SimplePropertyExpression) oof